### PR TITLE
Fix errcheck in ./common/membership/

### PR DESCRIPTION
**Pull request by MichaelSnowden**
**What changed?**
^

**Why?**
Fixing all errcheck warnings

**How did you test it?**
Reran errcheck

**Potential risks**
We may fail to start the GRPC resolver now when we can't add the listener

**Is hotfix candidate?**
No

### CODE REVIEW

File: common/membership/grpc_resolver.go [hunk 1]
```diff
@@ -131,7 +136,9 @@ func (m *grpcResolver) resolve() {
 			Addr: hostInfo.GetAddress(),
 		})
 	}
-	m.cc.UpdateState(resolver.State{Addresses: addresses})
+	if err := m.cc.UpdateState(resolver.State{Addresses: addresses}); err != nil {
+		fmt.Printf("error updating state in GRPC resolver: %v", err)
```
Comment (alexshtin) [inline common/membership/grpc_resolver.go:140]: pleeeeeese
```suggestion
		fmt.Printf("error updating state in gRPC resolver: %v", err)
```

Comment (alexshtin) [reply common/membership/grpc_resolver.go:140]: and bellow

**Review by alexshtin — APPROVED** [review]

File: common/membership/grpc_resolver.go (hunk 1 above)
Comment (MichaelSnowden) [reply common/membership/grpc_resolver.go]: Fixed!

File: common/membership/grpc_resolver.go [hunk 2]
```diff
@@ -131,7 +136,9 @@ func (m *grpcResolver) resolve() {
 			Addr: hostInfo.GetAddress(),
 		})
 	}
-	m.cc.UpdateState(resolver.State{Addresses: addresses})
+	if err := m.cc.UpdateState(resolver.State{Addresses: addresses}); err != nil {
+		fmt.Printf("error updating state in gRPC resolver: %v", err)
```
Comment (yycptt) [inline common/membership/grpc_resolver.go:140]: Use a logger?

File: common/membership/grpc_resolver.go [hunk 3]
```diff
@@ -142,7 +149,9 @@ func (m *grpcResolver) ResolveNow(_ resolver.ResolveNowOptions) {
 }
 
 func (m *grpcResolver) Close() {
-	m.r.RemoveListener(fmt.Sprintf("%p", m))
+	if err := m.r.RemoveListener(fmt.Sprintf("%p", m)); err != nil {
+		fmt.Printf("error removing listener from gRPC resolver: %v", err)
```
Comment (yycptt) [inline common/membership/grpc_resolver.go:153]: Use a logger?

**Pull request merged**